src/dal/interface/interface.spec.ts

```diff
@@ -1,6 +1,8 @@
 import { DbClient, IDbClient } from './index';
 import { MongoClient } from 'mongodb';
-import { db, close,collection, collections, dropCollection, createCollection } from '../../__mocks__/mongodb';
+import { db, close, collection, collections, dropCollection, createCollection } from '../../__mocks__/mongodb';
+
+jest.mock('mongodb');
```

Comment:
You can remove this line. We are doing a manual mock of `mongodb` so we don't have to import it here.